### PR TITLE
[pipelines] Add EB and History class API; Add example chat pipeline

### PR DESCRIPTION
**Pull request by sijunhe**

### CODE REVIEW

File: pipelines/pipelines/nodes/llm/ernie_bot.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import requests
+
+from pipelines.nodes.base import BaseComponent
+
+
+class ErnieBot(BaseComponent):
+    outgoing_edges = 1
+    headers = {"Content-Type": "application/json", "Accept": "application/json"}
+
+    def __init__(self, ernie_bot_access_token=None):
```
Comment (w5688414) [inline pipelines/pipelines/nodes/llm/ernie_bot.py:35]: """
        :param clean_header_footer: Use heuristic to remove footers and headers across different pages by searching
                                     for the longest common string. This heuristic uses exact matches and therefore
                                     works well for footers like "Copyright 2019 by XXX", but won't detect "Page 3 of 4"
                                     or similar.
"""
类似于这样加上注释

File: pipelines/pipelines/pipelines/base.py [hunk 2]
```diff
@@ -751,13 +754,11 @@ def load_from_config(
...
 
         pipeline = cls()
-        print(pipeline_definition)
         components: dict = {}  # instances of component objects.
         for node in pipeline_definition["nodes"]:
-            print("node", node)
             name = node["name"]
-            if name == "QAFilterPostprocessor":
-                print("exit")
+            # if name == "QAFilterPostprocessor":
```
Comment (w5688414) [inline pipelines/pipelines/pipelines/base.py]: 这两行注释的代码有用吗？

Comment (sijunhe) [reply pipelines/pipelines/pipelines/base.py]: 这个应该是你之前加的print, 用来debug的，要不然我直接删了？

Comment (w5688414) [reply pipelines/pipelines/pipelines/base.py]: 可以

**Review by w5688414 — APPROVED** [review]
LGTM

**Pull request merged**